web/src/components/MemoView.tsx

```diff
@@ -216,7 +216,7 @@ const MemoView: React.FC<Props> = (props: Props) => {
               <span className="btn more-action-btn">
                 <Icon.MoreVertical className="icon-img" />
               </span>
-              <div className="more-action-btns-wrapper">
+              <div className="more-action-btns-wrapper z-10">
```

Comment:
`z-10` is needed becuase it causes issues if there are preview cards in a memo on top and it makes the action menu not useable, in other words, cannot edit, delete a memo otherwise.

This makes the action menu appear on top of preview cards, hence allowing user to access the action menu